.github/workflows/create-docker-images.yml

```diff
@@ -0,0 +1,30 @@
+# Creates a new docker image and pushes it to DockerHub when a version tag is created.
+name: create-docker-images
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  # Based on instructions from https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
+  build-frontend-and-backend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
These lines are only necessary if we want to do multi-platform builds, which I don't think we do. 
```suggestion
```

Reply:
I am not familiar with the github worklow portion, but we want to target linux/amd64 platform while building, without these lines would it still work?